pkgs/desktops/gnome/extensions/guillotine/default.nix

```
@@ -0,0 +1,53 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitLab,
+  glib
```

Add a leading comma to reduce diff in case new inputs will be added in the future:

```suggestion
  glib,
```